The-New-Fork/openfood
# [Main] add function check network status and unittest

## Task
[JC-1909](https://thenewfork.atlassian.net/browse/JC-1909)

## Related Task
[JC-1907](https://thenewfork.atlassian.net/browse/JC-1907)

## Summary
add explorer_get_network_status function and unittest

## Changes
- add explorer_get_network_status function to check network status and return result in dictionary
- add unittest for explorer_get_network_status function

## Others (Optional)
How to run test :
- Go to **test folder** (/deploy-samples/simple-working/organization/blocknotify-python/openfood_lib_dev/tests)
- run command **pytest -s test_explorer_lib.py::test_explorer_get_network_status** (to check result from the test)
- run command **pytest -v test_explorer_lib.py::test_explorer_get_network_status** (to run test and to see that test has been passed)

If you want to run test for all function in test_explorer_lib.py, just use : **pytest -v test_explorer_lib.py**
